[cmake] remove usage of kodi-platform

olafhering opened the pull request:
The pvr.stalker binary addon does not use kodi-platform.
Remove existing references from CMakeLists.txt and debian/control.

Rechi commented on debian/control:
```diff
@@ -2,7 +2,7 @@ Source: kodi-pvr-stalker
 Maintainer: Nobody <nobody@kodi.tv>
 Build-Depends: debhelper (>= 9.0.0), cmake,
-               libkodiplatform-dev (>= 16.0.0), kodi-addon-dev,
+               kodi-addon-dev,
```
libp8-platform-dev is missing

olafhering commented:
Fixed

olafhering committed:
[cmake] remove usage of kodi-platform

The pvr.stalker binary addon does not use kodi-platform.
Remove existing references from CMakeLists.txt and debian/control.
Add dependency to libp8-platform-dev because libkodiplatform-dev
pulled that dependency in as a side effect. Fix that bug too by
adding an explicit dependency.

Signed-off-by: Olaf Hering <olaf@aepfle.de>
